vtex/faststore
# Chore: Add Component's Props

## What's the purpose of this pull request?

Adds missing props list on component's doc page.

- [x] RadioGroup *
- [x] RadioField
- [x] Overlay
- [x] Label
- [x] Link *
- [x] SROnly
- [x] PriceRange
- [x] InputField *
- [x] LinkButton *
- [x] Rating
- [x] Toggle
- [x] ToggleField


## How it works?

Updates our components documentation with list of props.
*= Added it manually because the automatization couldn't handle it properly.

## How to test it?

Go through the components' docs above and check if their props are listed as expected in the `Props` section.

[Documentation Preview Link](https://faststore-site-git-chore-add-components-props-fs-937-faststore.vercel.app/components/atoms/link)
